doc/installation.rst

```diff
@@ -94,13 +94,23 @@ the ``Dockerfile`` for the respective platform for what's needed.
       - :package-dev-deb:`DEB <docker_debian11>`
       - :repo:`Dockerfile <docker/Dockerfile.debian-11>`
 
+    * - Fedora 33
+      - :package-release-rpm:`RPM <linux_fedora33>`
+      -
+      -
+
+    * - Fedora 34
+      - :package-release-rpm:`RPM <linux_fedora34>`
+      -
+      -
+
     * - Fedora 35
-      - :package-release-rpm:`RPM <linux_fedora35>`
```

Comment:
Won't these be going live with the next release, and should hence be kept?

Reply:
Yes, they should, but are not yet available. I'll leave them in since we will cut a release soon.